library/modules/bigip_virtual_server.py

```diff
@@ -497,10 +497,20 @@
          - serverside
     type: list
     version_added: 2.8
+  check_profiles:
+    description:
+      - Specifies whether the client and server ssl profiles specified by the user should be verified to be
```

Comment:
break this down into separate sentences for this will look better in the docs

Reply:
fixed